### PR TITLE
Salt no longer blocks or stuns revenants.

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Salt now reveals revenants but does not block passage.
## Why It's Good For The Game
Salt is a trivial to make chemical while also being a direct upgrade from holy water when dealing with revenants. This PR makes holy water and the chaplain more useful against revenants by giving them different uses.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://user-images.githubusercontent.com/86113712/210021117-636743fd-87cf-4688-8118-4f70b7904e8e.mp4

</details>

## Changelog
:cl:
del: Salt no longer stuns or blocks revenants, they are revealed but can pass over it.
/:cl:

### CODE REVIEW

**Comment by Dejaku51** [comment]
ided

**Comment by benbot16** [comment]
A literal salt PR.
As someone who plays revenant a decent amount, I'd say this is unneeded. Salts incredibly easy to see compared to holy water, and you can easily remove it with defile (which any revenant should have anyways because morgue trays).

**Comment by WrillWasTaken** [comment]
The issue isn't that it's bad for the revenant, it's that people ignore the chaplain when salt is a direct upgrade. I wouldn't be opposed to making holy water stun instead in all honesty.

**Comment by St0rmC4st3r** [comment]
*slaps a 10000u water tank with a bible*
Trivial to make my ass!

**Comment by Tsar-Salat** [comment]
> Trivial to make my ass!

`/obj/item/storage/box/ingredients/ass
	theme_name = "st0rmc4st3r"`

**Comment by Zonespace27** [comment]
One could say this is a
*salt* PR

\*corny laughtrack\*

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
It should at least reveal them I think

**Comment by WrillWasTaken** [comment]
Updated PR name, description and testing evidence with changes.

File: code/modules/mob/mob_movement.dm [hunk 1]
```diff
@@ -246,13 +246,11 @@
 		if(INCORPOREAL_MOVE_JAUNT) //Incorporeal move, but blocked by holy-watered tiles and salt piles.
 			var/turf/open/floor/stepTurf = get_step_multiz(mobloc, direct)
 			if(stepTurf)
-				var/obj/effect/decal/cleanable/food/salt/salt = locate() in stepTurf
-				if(salt)
-					to_chat(L, "<span class='warning'>[salt] bars your passage!</span>")
+				if(locate(/obj/effect/decal/cleanable/food/salt, stepTurf))
+					to_chat(L, "<span class='warning'>The salt pile bars your passage!</span>")
```
Comment (EvilDragonfiend) [inline code/modules/mob/mob_movement.dm:250]: You're reverting the recent change https://github.com/BeeStation/BeeStation-Hornet/pull/8137
Please update your branch correctly.

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

**Comment by EvilDragonfiend** [comment]
Revealing time shouud be longer than before since rev can run away without being stunned. I think 5s will be enough.

**Comment by EvilDragonfiend** [comment]
Also, your change affects all ethereal movement type mobs like wiz

**Comment by WrillWasTaken** [comment]
>Also, your change affects all ethereal movement type mobs like wiz

Yes but also no
Salt still blocks ethereal movement types, revenants stop using ethereal movement when they get revealed.

**Comment by WrillWasTaken** [comment]
I have no idea how to unbungle this so I'm gonna close it until I figure out how.

**Pull request closed without merging**